Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -765,6 +716,95 @@ namespace AzToolsFramework
         }
     }
 
+    void DPERowWidget::AddColumnWidget(QWidget* columnWidget, size_t domIndex, const AZ::Dom::Value& domValue)
+    {
+        columnWidget->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Fixed);
+
+        // search for an existing column sibling with a lower dom index
+        int priorColumnIndex = -1;
+        for (int searchIndex = static_cast<int>(domIndex) - 1; (priorColumnIndex == -1 && searchIndex >= 0); --searchIndex)
+        {
+            priorColumnIndex = m_columnLayout->indexOf(m_domOrderedChildren[searchIndex]);
+        }
+
+        // if the alignment attribute is present, add the widget with its appropriate alignment to the column layout
+        auto alignment = AZ::Dpe::Nodes::PropertyEditor::Alignment.ExtractFromDomNode(domValue);
+        if (alignment.has_value())
+        {
+            Qt::Alignment widgetAlignment;
+            switch (alignment.value())
+            {
+            case AZ::Dpe::Nodes::PropertyEditor::Align::AlignLeft:
+                widgetAlignment = Qt::AlignLeft;
+                break;
+            case AZ::Dpe::Nodes::PropertyEditor::Align::AlignCenter:
+                widgetAlignment = Qt::AlignCenter;
+                break;
+            case AZ::Dpe::Nodes::PropertyEditor::Align::AlignRight:
+                widgetAlignment = Qt::AlignRight;
+                break;
+            }
+            m_columnLayout->WidgetAlignment(columnWidget, widgetAlignment);
+        }
+
+        //! If the sharePrior attribute is present, add the previous widget to the column layout.
+        //! Set the SharePrior boolean so we know to create a new shared column layout, or add to an existing one
+        auto sharePrior = AZ::Dpe::Nodes::PropertyEditor::SharePriorColumn.ExtractFromDomNode(domValue);
+        if (sharePrior.has_value() && sharePrior.value())
+        {
+            m_columnLayout->SharePriorColumn(m_columnLayout->itemAt(priorColumnIndex)->widget());
+            m_columnLayout->SetSharePrior(true);
+        }
+        else
+        {
+            m_columnLayout->SetSharePrior(false);
+        }
+
+        // If the UseMinimumWidth attribute is present, add the widget to set of widgets using their minimum width
+        auto minimumWidth = AZ::Dpe::Nodes::PropertyEditor::UseMinimumWidth.ExtractFromDomNode(domValue);
+        if (minimumWidth.has_value() && minimumWidth.value())
+        {
+            m_columnLayout->AddMinimumWidthWidget(columnWidget);
+        }
+
+        // insert after the found index; even if nothing were found and priorIndex is -1,
```

Comment:
uber nit: even if nothing *was* found

Reply:
ahh, but this is not a plurality case, the "were" is from often misused subjunctive mood! I respect the effort though.
https://www.rd.com/article/if-i-was-vs-if-i-were/

Reply:
I'll concede that you're technically grammatically correct and disagree and commit on this being idiomatic and helpful usage of subjunctive mood 😜